### ISSUE TITLE
Css conflict

### ISSUE THREAD

**Issue by IgorDePaula** [vuejs/vuepress#854]
#### Version
VuePress 0.14.2

#### Steps to reproduce

I put the AdminLite.io template in vue components, but this template use bootstrap. On doc this vue components with VuePress, the css from VuePress enter in conflict with bootstrap css.

https://igordepaula.github.io/AdminLTEVue/

#### What is expected?
Conflict None
#### Other relevant information

- Your OS:  Ubuntu 18.04
- Node.js version:  8.10.0
- Browser version: Firefox 64 (64 bits)
- Is this a global or local install? 
Global, but local install is fat the components.
- Which package manager did you use for the install? 
NPM

**Comment by IgorDePaula** [comment]
I dont understand

**Comment by IgorDePaula** [comment]
@Deraw- You suggest me create a new theme for vuepress?

**Comment by IgorDePaula** [comment]
=/ , I appreciate your attention

**Comment by mgh87** [comment]
Is there any solution for vuepress with vuetify that does not completely wrack the default theme?

**Comment by Deraw-** [comment]
@mgh87 Nope! I mean it's like you wanted to use both materialize & bootstrap at the same time, you would have conflicts
Maybe you can create a Vuepress theme that uses Vuetify and keeps the same structure as the default theme!

**Comment by mgh87** [comment]
@Deraw- Well, my approach would be to use vuetify css only within its components. I tried to achieve this in creating own single file components with scoped css for arrangements where I use vuetify, but this does not work. After a loaded a page with a vuetify component, the css gets overwritten and is used for further navigation through the page.

I currently try to use just the bits of stylus definitions from vuetify that is required for the used components to render proper, but that is a rather hacky approach and will just increase maintenance cost. Is there any better approach to solve this?

**Comment by Deraw-** [comment]
@mgh87 Yes that's normal, Vuetify has global styles in it and will register them in `<head>`
I'll try to create a Vuepress + Vuetify theme

**Comment by mgh87** [comment]
@Deraw- we currently fixed the problem with the styles by restricting which styles we add to our components, that works rather good. So there is no urgent need for us to have a theme.
Ty for the explanation where the injection is done.

**Comment by ibdf** [comment]
Anyone else came up with a creative solution to this problem? Vuepress seems like a great solution to create documentation but when rendering components there's so much conflict between the component css being rendered and the theme's css.

I tried creating a component with a <style scoped> with all the css I would need in it, and then loading all other components through via its slot, but it looks like the scoped option is not respected, so even if I only import the css in a component once the the component renders, then everything breaks.

There's no way I can manually scope it, as I am using a framework.
